src/ADRIA.jl

```diff
@@ -103,7 +103,7 @@ export ReefModDomain
 # export dims, ndims
 
 # List out compatible domain datapackages
-const COMPAT_DPKG = ["0.3.1", "0.4.0-rc", "0.4.0"]
+const COMPAT_DPKG = ["0.4.0-rc", "0.4.0", "0.5.0-rc", "0.5.0"]
```

Comment:
Is a new Moore datapackage on the DS store?

Reply:
Not yet, I'll submit it later this afternoon. 

But if you make a copy and make the changes as outlined it'll work.

Reply:
Oh wait, I've got to push some changes for `ReefModDomain`

Reply:
Thanks, I'm just running tests now but I'll run them again when you've pushed

Reply:
It won't matter, we don't have any tests covering the ReefModDomain because we don't want to commit a copy of that dataset into this repo.

We could mock up an example, but I don't have time for that today. Do you want to have a crack at it?

Reply:
Yeah sure, I can do that. Just a test for loading the domain and running test scenarios?

Reply:
Yes, it's just the test domain restructured to match how RME's file structure

Reply:
Tests are all passing, there's just a warning being thrown because the test domain still has `UNIQUE_ID` in the site data instead of `cluster_id`. I can change this when I add a test domain for ReefMod if that sounds good?

Reply:
Rose, that warning is there on purpose. It's because it can't find `cluster_id`, not because `UNIQUE_iD` is in the geopackage

Reply:
Yes, so should I add a `cluster_id` to the test set?

Reply:
> The reason I said "instead of" was that UNIQUE_ID was originally used for spatially constraining selected sites

What I'm reacting to is this:

> a warning being thrown *because* the test domain *still* has UNIQUE_ID in the site data *instead of* cluster_id.

This is expected behaviour. If `cluster_id` cannot be found, it will use `UNIQUE_ID`.
By saying "still" and "instead of", you seem to be suggesting that this is incorrect unexpected behaviour and that `UNIQUE_ID` should be removed.

> should I add a cluster_id to the test set?

Rather than adding `cluster_id`, perhaps it would be nicer if a test is added to ensure that the above warning is thrown?
I'll leave you to do it in a separate PR.

Reply:
I understand this is expected behaviour, my point was that the test domain should have the preferred id column so that you don't get heaps of warnings when running tests